source/renderer/app/i18n/locales/defaultMessages.json

```diff
@@ -4635,7 +4635,7 @@
         }
```

Comment:
Someone forgot to run `yarn manage:translations` and it was merged to develop.

Reply:
It means that our CI didn't catch that :/

Reply:
It must've been before my CI fix was merged